src/styles/blocks/cards.scss

```
@@ -0,0 +1,107 @@
+.cards {
+  width: maxWidth($card-size,$margin-size);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  padding: 50px 0;
+
+  &__card {
+    margin-right: $margin-size;
+    margin-bottom: 46px;
+  }
+
+  &__card:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__card:nth-child(5),
+  &__card:nth-child(6),
+  &__card:nth-child(7),
+  &__card:nth-child(8) {
```

```suggestion
  &__card::nth-last-child(-n + 4) {
```
Its in the requirements "use :nth-last-child(-n + 4) to select 4 last elements"